### PR TITLE
Make Code Of Conduct, Readme and setup.py consistent with other docs

### PR DESCRIPTION
**Pull request by Julian-O**
These documents duplicate information stored elsewhere. Bring them up-to-date:

* Change CODE_OF_CONDUCT.md to to point to Contribution Guidelines.

* Make Kivy tag line in README consistent with FAQ and Discussions.

* Change README to point to Contact us and Contributions pages.

* Also, add a few more useful badges, and tidy them up.

* Make setup.py/PyPI description consistent with new README.

* Minor copy-edits. Mention Raspberry Pi. Gender-neutral language, Stop mentioning OpenGL ES 2.

Note: Some links are to MASTER documents, not STABLE, where they are not linked to releases and may change faster than that.

Maintainer merge checklist
* [ ] Title is descriptive/clear for inclusion in release notes.
* [ ] Applied a `Component: xxx` label.
* [ ] Applied the `api-deprecation` or `api-break` label.
* [ ] Applied the `release-highlight` label to be highlighted in release notes.
* [ ] Added to the milestone version it was merged into.
* [ ] **Unittests** are included in PR.
* [ ] Properly documented, including `versionadded`, `versionchanged` as needed.

### CODE REVIEW

**Comment by Julian-O** [comment]
Note to self: When this is updated, Buildozer README is next.

File: README.md [hunk 1]
```diff
@@ -105,9 +106,9 @@ Licenses
   and is licensed under the terms of the
   [LGPLv2.1](https://www.gnu.org/licenses/old-licenses/lgpl-2.1).
 
-## Contributors
+## Financial Contributors
 
-This project exists thanks to all the people who contribute. [[Contribute](.github/CONTRIBUTING.md)].
+This project exists thanks to all the people who donate. [[Donate](.github/CONTRIBUTING.md)].
```
Comment (misl6) [inline README.md:111]: `.github/CONTRIBUTING.md` is not just about financial contributions.
And IMHO should not.

`https://opencollective.com/kivy/contributors.svg` is just showing up all our Github (code) contributors.

Instead, below we have the backers one, dedicated to OpenCollective (financial) backers.

**Review by misl6 — CHANGES_REQUESTED** [review]
Hi @Julian-O !

I've left a review, let me know what do you think about it.

File: README.md [hunk 2]
```diff
@@ -3,29 +3,34 @@ Kivy
 
 <img align="right" height="256" src="https://raw.githubusercontent.com/kivy/kivy/master/kivy/data/logo/kivy-icon-256.png"/>
 
-Innovative user interfaces made easy.
+[Kivy](http://kivy.org) is an open-source Python framework for developing
```
[the remark this answers is not in the document]
Comment (Julian-O) [reply README.md:6]: Done.

File: README.md [hunk 3]
```diff
@@ -3,29 +3,34 @@ Kivy
...
 The aim is to allow for quick and easy interaction design and rapid prototyping
-whilst making your code reusable and deployable.
+whilst making your code reusable and deployable: Innovative user interfaces made
+easy.
 
-Kivy is written in Python and [Cython](http://cython.org/), based on OpenGL ES
-2, supports various input devices and has an extensive widget library. With the
-same codebase, you can target Windows, macOS, Linux, Android and iOS. All Kivy
-widgets are built with multitouch support.
+Kivy is written in Python and [Cython](http://cython.org/), supports various input devices and
```
[the remark this answers is not in the document]
Comment (Julian-O) [reply README.md:14]: Done

File: README.md [hunk 4]
```diff
@@ -3,29 +3,34 @@ Kivy
...
+has an extensive (and extensible) widget library. With the
+same codebase, you can target Windows, OSX, Linux, Android, iOS and Raspberry
```
[the remark this answers is not in the document]
Comment (Julian-O) [reply README.md:16]: Ohhh! That explains the inconsistency. Done, but I not the names of the workflows are harder to change so I left as OSX.

File: README.md (hunk 1 above)
Comment (Julian-O) [reply README.md:111]: Ahh. I see my misunderstanding. Sorry.

Done.

We have yet another duplicate of the same information. In the *other* review branch (about the FAQ and `docs/Contributing.rst`) I will update `.github/CONTRIBUTING.md` to also be in sync.

**Comment by Julian-O** [comment]
Note to self: When this is done, extend the sponsor links, so it isn't just the first 10.

File: README.md [hunk 5]
```diff
@@ -3,29 +3,34 @@ Kivy
...
-Kivy is an open source, cross-platform [Python](https://www.python.org)
-framework for the development of applications that make use of innovative,
-multi-touch user interfaces.
 The aim is to allow for quick and easy interaction design and rapid prototyping
-whilst making your code reusable and deployable.
+whilst making your code reusable and deployable: Innovative user interfaces made
+easy.
 
-Kivy is written in Python and [Cython](http://cython.org/), based on OpenGL ES
-2, supports various input devices and has an extensive widget library. With the
```
Comment (mak8kammerer) [inline README.md:15]: I think we shouldn't delete the mention of OpenGL ES 2. This may be important for developers.

**Comment by Julian-O** [comment]
Given I was making another edit, I went ahead and made a third - getting the sponsors logos to appear.

**Review by mak8kammerer — APPROVED** [review]

**Comment by misl6** [comment]
Don't hate me! 😄 I'm just exploring the best approach for managing the contribution guide, code of conduct, and potential FAQs.

While reviewing the axios.js Docs at (https://axios-http.com/docs/intro) today, I was inspired by the idea of incorporating links that navigate directly to relevant markdown files hosted on GitHub, see here:

Image 16-11-23 at 18 31


One potential concern with linking to the master version of the docs is that users might overlook the branch selection, unintentionally accessing a different version of the documentation. This could lead to confusion, so we should carefully consider how to address this potential issue.

What are your thoughts?

**Comment by Julian-O** [comment]
> Don't hate me! 😄

No fear. Recently, I have been working on projects with me as the solo developer. I was surprised to find I *miss* having code reviews, and having other people care about the miniscule design decisions I made. 

What I am hearing here is:
* Agreement that we shouldn't repeat ourselves in two places.
* Agreement that, with some documents (e.g. API descriptions) it is important to have the documentation matching the installed version. Other times (e.g. Contact us details) it is important to have the latest.
* Concern that this distinction is hard to maintain when in the Kivy documentation pages - the user might be sent to one page and then navigate to inappropriately versioned information.

So the suggestion is: Where the information should always be the latest rather than versioned, put the master copies in GitHub Markdown pages and reference that from the RSTs, rather than the other way around.

Okay.

* I will also look at some sort of reminders when people are being sent to Kivy.org/docs.
* I will need to revisit https://github.com/kivy/python-for-android/pull/2922
* I am starting to second guess whether this should be in the Kivy.org HTML instead.

**Comment by misl6** [comment]
> I am starting to second guess whether this should be in the Kivy.org HTML instead

If we need to have something on the Kivy website (e.g. a searchable FAQ, categorized for each subproject), we can build a bot to update the website.

**Comment by Julian-O** [comment]
There were two PRs that were starting to overlap, so I have pushed the contents of this one into #8448

**Pull request closed without merging**